addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1390,6 +1391,368 @@ describe('Paste', () => {
                     contentAfter: '<ul><li>123</li><li>456[]abc</li><li>def</li><li>ghi</li></ul>',
                 });
             });
+            it('should paste a nested list into another list', async () => {
+                await testEditor(BasicEditor, {
+                    contentBefore: '<ol><li>Alpha</li><li>Beta</li><li>[]</li></ol>',
+                    stepFunction: async editor => {
+                        await pasteHtml(editor, unformat(`
+                            <ul>
+                                <li>abc</li>
+                                <li>def</li>
+                                <li class="oe-nested">
+                                    <ul>
+                                        <li>123</li>
+                                        <li>456</li>
+                                    </ul>
+                                </li>
+                            </ul>
+                        `));
+                    },
+                    contentAfter: unformat(`
+                        <ol>
+                            <li>Alpha</li>
+                            <li>Beta</li>
+                            <li>abc</li>
+                            <li>def</li>
+                            <li class="oe-nested">
+                                <ol>
+                                    <li>123</li>
+                                    <li>456[]</li>
+                                </ol>
+                            </li>
+                        </ol>
+                    `),
+                });
+            });
+            it('should paste a nested list into another list', async () => {
+                //TODO: change this test case after fixing copy paste in empty node (task-3302332)
+                await testEditor(BasicEditor, {
+                    contentBefore: '<ul><br><li>Alpha</li><li>Beta</li><li>[]</li></ul>',
```

Comment:
Equivalent comment as above. Needs a `<br>`.
But please remove the `<br>` before the first `<li>`. Why is is there?

Reply:
:confused: @shsa-odoo you added this could you please enlighten us here.

Reply:
@Zynton indeed that <br> serves no purpose, @deso-odoo  while you are at it , remove that as well. Thanks